pkg/transcriptionsystems/bipa/consonants.tsv

```diff
@@ -606,3 +609,10 @@ kʲ	voiceless	velar	stop		palatalization:palatalized
 k̚	voiceless	velar	stop		release:unreleased	
 kʷʼ	voiceless	velar	stop	+	labialization:labialized,ejection:ejective	
 ɡ̤	voiced	velar	stop	+	breathiness:breathy	
+d̪l̪	voiced	lateral	affricate			
+ɽr	voiced	retroflex	trill			
+kp	voiceless	labio-velar	stop			
```

Comment:
This is what we HAD handled as a double-articulation. There's no IPA rule that would say that kp is a labiovelar. A labiovelar is what we denote as [k^w](https://clts.clld.org/parameters/labialized_voiceless_velar_stop_consonant).

Reply:
/kp/ in IPA would be a cluster, which is how it is parsed now. The labio-velar has the bar on top, which we don't use in BIPA; given that "kp" as a cluster will surely be less common a single segment than the labio-velar, I presented it as the latter.

I am not sure I understand "k^w", you mean a superscript "w" as in kʷ? It would be the labialized plosive.